app/models/host/managed.rb

```diff
@@ -320,10 +320,10 @@ class Jail < ::Safemode::Jail
                         :if => proc { |host| host.managed && host.disk.empty? && !Foreman.in_rake? && !host.image_build? && host.build? }
   validates :provision_method, :inclusion => {:in => proc { provision_methods }, :message => N_('is unknown')}, :if => proc { |host| host.managed? }
   validates :medium_id, :presence => true,
-                        :if => proc { |host| host.validate_media? }
+                        :if => proc { |host| host.managed? && host.validate_media? }
```

Comment:
Isn't this redundant? `validate_media?` already checks for `managed`:
https://github.com/theforeman/foreman/blob/e78bd159584f2f8d09b3d57ab94615a2ae26ecb7/app/models/host/managed.rb#L747-L749

Reply:
Thanks @ekohl Good catch. I was actually mainly testing https://github.com/theforeman/foreman/pull/10226/files#diff-b7f82bd4fe3d5a5a8b246f502ab1e894197164cf7bf48fbfd5c2f3d76da2362eL326 and thought its  also a good idea to have the `managed?` check for this part, too.